backend/services/notification_enqueue_service.py

```
@@ -262,3 +262,74 @@ def _get_push_tokens_for_users(self, user_ids: List[str]) -> List[str]:
+            
+            if success:
+                logger.info(
+                    f"Monthly dump batch notification enqueued for {month}: recipients={len(push_tokens)}"
+                )
+            else:
+                logger.error(f"Failed to enqueue monthly dump batch notification for {month}")
+                
+            return success
+            
+        except Exception as e:
+            logger.error(f"Error enqueueing monthly dump batch notification for {month}: {str(e)}", exc_info=True)
```

Ruff (0.15.11)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)